### ISSUE TITLE
[LEGEND] Redesign the legend

### ISSUE THREAD

**Issue by zufanka** [tenders-exposed/elvis-ember#446]
Split legend into multiple pictures to use with the carousel.

**Comment by zufanka** [comment]
Related https://github.com/tenders-exposed/elvis-ember/issues/447

**Comment by zufanka** [comment]
1. explain colours: company and government dots
2. explain sizes of the dots
"Size of the company dot shows how much money (or tenders) this company got from all the governments in the network visualisation" 
"Size of the government dot shows how much money (or tenders) this government awarded to all the companies in the network visualisation" 
3. explain connections and size of connections
"The connections show how much money (or tenders) these two actors exchanged within this particular network" 
4. explain clustering

**Comment by chutko** [comment]
here are the slides for the tutorial carousel + network loader (and loaded state) 
i haven't made the clustering slide, im a bit confused about what to put there

![elvis--tutorial-company-loading](https://user-images.githubusercontent.com/4135015/44585057-45608600-a7ab-11e8-962f-0029f8dc41dd.png)
![elvis--tutorial-company-loaded](https://user-images.githubusercontent.com/4135015/44585056-45608600-a7ab-11e8-9503-5005cde729a5.png)
![elvis--tutorial-government](https://user-images.githubusercontent.com/4135015/44585058-45f91c80-a7ab-11e8-9343-21e24cd70bdb.png)
![elvis--tutorial-relationship](https://user-images.githubusercontent.com/4135015/44585059-45f91c80-a7ab-11e8-9848-0409e5f2d63f.png)

**Comment by georgiana-b** [comment]
This is not accurate since we don't show number of _tenders_, we show number of _bids_ won. So a company can win 2 bids in the same tender in which case we will show 2.

It depends how accurate you want to be at this stage but further down  in the details view people will encouter both concepts and they might get very cofused. So I think we could actually take the opportunity of the carousel ride to explain `tenders`, `lots` and `bids` as well. 

@zufanka @chutko WDYT about :point_up: ?

So this way we would end up with texts like:

> Size of the Government dot shows how much money or _how many bids_  this _institution_ awarded  towards all the Companies in the network.

In the Relationship view:

> The thickness of the line connecting a Government and Company indicates the amount of money or number of bids awarded by the Government to that Company.

**Comment by chutko** [comment]
i think you are right, although right now i would like to implement what we have (with content changes you suggest above), it will not do any harm and things will get done on time which is this week :)

and along with that, we could discuss content of other slides and put them on when we got them.

**Comment by zufanka** [comment]
I think "bids" is technical lingo. I would rather use "(parts of) tenders" or "tenders (or parts of tenders)" or something like that. @georgiana-b @chutko

**Comment by chutko** [comment]
but we are using this technical lingo, **Bids** inside the network. so we should stay consistent, shouldn't we?

**Comment by chutko** [comment]
okay guys, if you all agree, this is the final export ready for the carousel (ping @ca1yps0). i have exported it in double size (1380x660) so it looks good when resized on mobile later (when actually something works on mobile). for now, the size used in the carousel should be 690px x 330px

![elvis--tutorial-company](https://user-images.githubusercontent.com/4135015/44724786-0a749000-aad4-11e8-8dc0-17564facd010.png)
![elvis--tutorial-institution](https://user-images.githubusercontent.com/4135015/44724788-0a749000-aad4-11e8-9edc-4addbbcc4bd7.png)
![elvis--tutorial-relationship](https://user-images.githubusercontent.com/4135015/44724789-0b0d2680-aad4-11e8-9674-073fcd322b7a.png)

**Comment by georgiana-b** [comment]
A public procurement tender is a call to offers initiated by an institution when they
Companies who are willing to supply the services or goods desired by the institution bid for the tender.

**Comment by georgiana-b** [comment]
A public procurement tender is a call to offers initiated by an institution when it needs to purchase goods or services.
Companies who are willing to supply the desired goods bid for the tender. Eventually the institution chooses a winning bid and awards the tender to the company who created that bid (offer).

Some tenders involve, for example, different locations and they are split into multiple lots so that each lot gets the most suitable offer for its specific circumstances. A company can bid for multiple lots and have multiple winning bids in the same tender.

So in the network we count how many bids have been awarded by an institution to a company. For the sums, we add up the amount of money awarded to the company in each of its winning bids.

**Comment by georgiana-b** [comment]
@chutko  I know :point_up: is too long for one card but I really can't see how to fit all the explanations in a phrase. 
Could maybe split it in 2 cards?

The card you made during the call today would work great with this part:

> Some tenders involve, for example, different locations and they are split into multiple lots so each lot gets the most suitable offer for its specific circumstances. A company can bid for multiple lots and have multiple winning bids in the same tender.
So in the network we count how many bids have been awarded by an institution to a company. For the sums, we add up the amount of money awarded to the company in each of its winning bids.

Then maybe we can have another card that represents the tender and the bids that are opened by different companies spread around it, for the first part:

> A public procurement tender is a call to offers initiated by an institution when it needs to purchase goods or services.
Companies who are willing to supply the desired goods bid for the tender. Eventually the institution chooses a winning bid and awards the tender to the company who created that bid (offer).

**Comment by georgiana-b** [comment]
Also if anyone can come up with an even more compressed yet still understandable tender description please do :smiley_cat: .

**Comment by zufanka** [comment]
@chutko hereby the texts amendments for existing cards:
* _Company_ : Size of the company dot shows how much money **(or winning bids)** this company got from all the governments in the network visualisation
* _Government_:  Size of the company dot shows how much money **(or winning bids)** this **government** institution awarded **to** all the companies in the network visualisation
- _big dot bubble_: We **have** awarded quite some money **(or winning bids) in this network**
* _Relationship_: Thickness of **the** connecting line between companies and government **dots** shows the amount of money  **(or winning bids)** awarded **by** this government **to** this company 
- _big dot bubble_: Yep, there has been **a lot of business between us, a lot of €€€ (or winning bids). Might be suspicious?**

@georgiana-b good content, but it is quite the mouthful :) What about this? 

* A tender can be split into multiple parts. For example cleaning services for buildings of the same government institution in different towns. Or hardware, software and maintenance of computers can be three different parts of the same tender. Each part can then be won by one or multiple companies. This is what we call a "winning bid". Therefore one tender can have one or multiple parts and also winning bids.

OR even _shorter_, as I think we don't need to explain 'lots' at all:

* A tender can be won by multiple companies. For example cleaning services for buildings of the same government institution in different towns. Therefore one tender can have one or more "winning bids".

@chutko for the _clustering_:

* Sometimes one company appears as multiple dots in the network. You can join dots by using the "merge" button in the right menu.

**Comment by georgiana-b** [comment]
@zufanka @chutko FYC, we do show `Lots` in Elvis:

![image](https://user-images.githubusercontent.com/8862002/44784399-38baa400-ab96-11e8-8f0f-b87417cbd2c9.png)

**Comment by zufanka** [comment]
@georgiana-b oh yea, crap ^^
Aight, how about this?

* A tender can be split into multiple parts (lots). For example cleaning services for buildings of the same government institution in different towns. Each part can then be won by one or multiple companies. Therefore one tender can have one or multiple parts and also winning bids.

**Comment by georgiana-b** [comment]
This is the shortest version I can come up with :sweat_smile: 

> An institution creates a tender - call to offers - when it wants to purchase goods or services.
Sometimes the tender is split into multiple lots, corresponding to different buildings of the institution, for example.

> Companies bid - make offers - to the lots they are willing to supply for. The institution then chooses a winning bid for each lot. A company can have multiple winning bids in the same tender.

**Comment by georgiana-b** [comment]
@zufanka I started formulating the above before seeing your comment but I really wanted to  exmplain tenders as well.

**Comment by zufanka** [comment]
@georgiana-b 

* _corresponding to different buildings of the institution, for example._ -- it's very unclear what this means. If we want to use it, we should use it as a full example.

* _A company can have multiple winning bids in the same tender._ -- what do you mean? Isn't it about a tender and lots having multiple winning bids?

**Comment by georgiana-b** [comment]
@zufanka You are right `corresponding to different buildings of the institution, for example` is unclear by itself but I formulated it like that for the sake of brevity. I thought the next phrase will clarify:

> Sometimes the tender is split into multiple lots, corresponding to different buildings of the institution, for example.
Companies bid - make offers - to the lots they are willing to supply for.

If you still find it unclear let's just put in your full example:

> The tender can be split into multiple lots. For example cleaning services for buildings of the same government institution in different towns.

As for your second point, in the first paragraph we explain that tenders have multiple lots. Then we explain that each lot can have bids (offers) from multiple companies. Then we say each lot gets a winner. In that last sentence I just wanted to point out there is no rule stopping a company from winning multiple lots in the same tender.

**Comment by zufanka** [comment]
ok, let me see. I think we have lost the goal here on the way :)
The purpose of this slide was to explain **what we actually count** when we count tenders. We don't really count tenders, we count winning bids. So we want to say:

- What is a tender
- Tender can have multiple parts, these are called lots.
- Lots can have multiple winners, that is what we call "winning bids" and that is what we count

Let's just formulate it simply and maybe show the text to other people so they can tell us whether it makes sense

**Comment by georgiana-b** [comment]
An institution creates a tender - call to offers - when it wants to purchase goods or services.
Sometimes the tender is split into multiple lots, corresponding to different buildings of the institution, for example.

Companies bid - make offers - to the lots they are willing to supply for. The institution then chooses a winner for each lot. So in the network we count how many winning bids a company had in business with an institution or how much money was awarded to the company though these winning bids.

**Comment by zufanka** [comment]
Ok! I would leave out the "corresponding to different buildings of the institution, for example." .. it's brief but still confusing :)

The last sentence I would rephrase to:
"The network shows how many bids the company won or how much money was awarded to it though these winning bids."

**Comment by georgiana-b** [comment]
I think it's ok to leave out that part.

As for:
 
> "The network shows how many bids the company won or how much money was awarded to it though these winning bids."

I was also tempted to use  `how many bids the company won` but if you think about it it doesn't really make sense. We just got into the habit of using it so makes sense to us but in fact the _the company wins lots though bids_, the company doesn't win bids, it authors them. 

So  we say something like `we count how many lots the company won` or `how many of the company's bids won` or `how many winning bids the company has`. 

The part with `how much money was awarded to it though these winning bids.` sounds good.

**Comment by zufanka** [comment]
Good point @georgiana-b !
I have just added two sentences of clarifications at the end of both parts, are we good to go with this description?

* An institution creates a tender - call to offers - when it wants to purchase goods or services.
Sometimes the tender is split into multiple lots. These can have one or more winners.
The network shows how many of the company's bids won or how much money was awarded to it though these winning bids. One tender can therefore contain multiple winning companies.

@chutko you were right, it really was difficult ^^

**Comment by chutko** [comment]
> An institution creates a tender - call to offers - when it wants to purchase goods or services.
> Sometimes the tender is split into multiple lots. These can have one or more winners.
> The network shows how many of the company's bids won or how much money was awarded to it though these winning bids. One tender can therefore contain multiple winning companies.

its still too long and doesnt fit into the space, sorry :D

**Comment by zufanka** [comment]
@chutko Lose the last sentence?

**Comment by chutko** [comment]
okay, here is the (hopefully) final export of the slides. all these images are exported in double size, the size of the carousel image on desktops should be 690 x 330px

![elvis--tutorial-tender](https://user-images.githubusercontent.com/4135015/44945980-6ea89400-adf3-11e8-8984-02c0af131946.jpg)
![elvis--tutorial-merging](https://user-images.githubusercontent.com/4135015/44945981-6ea89400-adf3-11e8-9b93-aa8a5d03eb60.jpg)
![elvis--tutorial-relationship](https://user-images.githubusercontent.com/4135015/44945982-6ea89400-adf3-11e8-9292-2bfd9f3ec80d.jpg)
![elvis--tutorial-government](https://user-images.githubusercontent.com/4135015/44945983-6ea89400-adf3-11e8-9ce9-378033af8630.jpg)
![elvis--tutorial-company](https://user-images.githubusercontent.com/4135015/44945984-6ea89400-adf3-11e8-8fde-3392be2bf519.jpg)

@ca1yps0

**Comment by georgiana-b** [comment]
These are all looking good now so I'll close this.
